Clinical-Genomics/scout
# Create a case-level event when a user resets all dismissed variants

This PR adds a functionality:
-  fix #2416 
- Makes sure that the activity events are displayed with the newest first on case and variant pages

**How to test**:
1. Dismiss one or more variants for a page
2. In master branch, reset all dismissed variants from one of the variantS page
3. Make sure that the reset dismissed events are visible only on the involved variants pages
4. Switch to this branch and repeat. It should now display that your specific user did the reset of all dismissed variants

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
